src/views/tutorial-view/components/next-previous/components/directional-link-box/directional-link-box.module.css

```diff
@@ -1,101 +1,36 @@
-.card {
-  border: 1px solid magenta; /** @TODO style these to spec, remove any prev imp styles necessary - https://www.figma.com/file/VD7ahvXuXWJApeGnhbW4hv/Dev-Portal?node-id=6298%3A73041 */
-  position: relative;
-  display: block;
-  background: var(--white);
-  border-radius: 5px;
-  margin: 0;
-  padding: 20px;
-}
-
-/* visually hidden */
-.hidden {
-  clip: rect(0 0 0 0);
-  clip-path: inset(50%);
-  height: 1px;
-  overflow: hidden;
-  position: absolute;
-  white-space: nowrap;
-  width: 1px;
-}
-
 .linkbox {
-  composes: card;
   display: flex;
+  flex-wrap: wrap;
   align-items: center;
-  width: 100%;
-  background-color: var(--white);
-  color: var(--black);
-  box-shadow: var(--token-surface-mid-box-shadow);
-  transition: box-shadow 0.3s;
-
-  &[data-direction='previous'] {
-    text-align: left;
-    justify-content: flex-start;
-    &:hover {
-      box-shadow: var(--token-surface-high-box-shadow);
-    }
-  }
-
-  &[data-direction='next'],
-  &[data-direction='final'] {
-    text-align: right;
-    justify-content: flex-end;
-
-    &:hover {
-      box-shadow: var(--token-surface-high-box-shadow);
+  text-align: center;
+  justify-content: center;
+  height: 100%;
+
+  @media (--medium-up) {
+    &.direction-previous {
```

Comment:
Could you update these classnames to be `camelCase` instead of `kebab-case`?

Reply:
I can try, but the incoming props are in lower case (`next`, `previous`, `final`). I opted for kebab-case since otherwise I think I'd have to mess around with capitalization. Let me know if that seems alright! ☺️

Reply:
☝️ I suppose I could change the class names to be `.next` , `.previous`, & `.final` which would technically be `camelCase`, with no need to capitalization mess-arounds. I have a slight preference for the explicitness of prefixing with `direction` (matches the prop nicely, makes it a little more transparent), but very slight, so happy to adjust to whatever feels best for everyone else.

Reply:
Ohhh I didn't notice that! Then `kebab-case` makes total sense here.